core/templates/dev/head/pages/topics_and_skills_dashboard/TopicsListDirective.js

```diff
@@ -62,8 +81,11 @@ oppia.directive('topicsList', [
 
             modalInstance.result.then(function() {
               EditableTopicBackendApiService.deleteTopic(topicId).then(
-                function() {
-                  $rootScope.$broadcast(EVENT_TOPIC_DELETED, topicId);
+                function(status) {
+                  if (status === 200) {
```

Comment:
This doesn't look complete. What are the other possible cases? What do we do if they occur?

Reply:
Wouldn't the error callback written below this be executed if an error occurs?

Reply:
Removing the check sounds fine and addresses my concern, thanks!